core/templates/pages/exploration-player-page/exploration-player-page.mainpage.html

```diff
@@ -39,6 +39,7 @@
     @load('pages/footer_js_libs.html')
     <script src="/third_party/static/ckeditor-4.12.1/ckeditor.js"></script>
     <script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+    <script async src="/third_party/static/MathJax-2.7.5/jax/output/HTML-CSS/jax.js?V=2.7.5"></script>
```

Comment:
One question. Why don't the other script tags have the "async" attribute?

Reply:
I have reworked this, no longer applies.